### PR TITLE
remove test models and fix order of checks

### PR DESCRIPTION
**Pull request by eaidova**
* added removing test models after llm bench tests passing to reduce disk space
* fixed order of checks in wwb tests (csv file is result of successful execution of cli command, test trying to open file and only after that check cli command return code)
* reduces inference counts in llm bench tests (reduces execution time in 2 times, from 72 min to 36 min)

### CODE REVIEW

File: .github/workflows/llm_bench-python.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ jobs:
       with:
         platform: ubuntu22
         commit_packages_to_provide: wheels
-        revision: latest_available_commit
+        revision: 747d0e7e105c9f2c9966a37861f95b1c7f886868
```
Comment (eaidova) [inline .github/workflows/llm_bench-python.yml:47]:
```suggestion
        revision: latest_available_commit
```

File: .github/workflows/llm_bench-python.yml [hunk 2]
```diff
@@ -143,6 +143,7 @@ jobs:
           optimum-cli export openvino --model OpenGVLab/InternVL2-1B ./ov_models/internvl2-1B --task image-text-to-text --trust-remote-code
           python ./tools/llm_bench/benchmark.py -m ./ov_models/internvl2-1B --media https://github.com/openvinotoolkit/openvino_notebooks/assets/29454499/d5fbbd1a-d484-415c-88cb-9986625b7b11 --prompt "What is unusual on this image?" -ic 20
           python ./tools/llm_bench/benchmark.py -m ./ov_models/internvl2-1B --media https://github.com/openvinotoolkit/openvino_notebooks/assets/29454499/d5fbbd1a-d484-415c-88cb-9986625b7b11 --prompt "What is unusual on this image?" -ic 20 --optimum
+          rm -rf ./ov_models/internvl2-1B
```
Comment (ilya-lavrenov) [inline .github/workflows/llm_bench-python.yml:146]: should we delete other models in steps above?

Comment (eaidova) [reply .github/workflows/llm_bench-python.yml:146]: yes, for now I deleted the heaviest for the experiment if it helps then I'll update to remove other models too

**Review by ilya-lavrenov — APPROVED** [review]

File: .github/workflows/llm_bench-python.yml (hunk 2 above)
Comment (eaidova) [reply .github/workflows/llm_bench-python.yml:151]: other models also removed

File: .github/workflows/llm_bench-python.yml [hunk 3]
```diff
@@ -34,7 +34,7 @@ jobs:
     runs-on: aks-linux-2-cores-8gb
     container:
       image: 'openvinogithubactions.azurecr.io/openvino_provider:0.1.0'
-      volumes:
+      volumes:
```
Comment (eaidova) [inline .github/workflows/llm_bench-python.yml:37]:
```suggestion
      volumes:
```

File: .github/workflows/llm_bench-python.yml [hunk 4]
```diff
@@ -104,29 +103,33 @@ jobs:
         run: |
           git clone --depth 1 https://huggingface.co/katuni4ka/tiny-random-qwen
           python ./tools/llm_bench/benchmark.py -m tiny-random-qwen -d cpu -n 1 -f pt
```
Comment (eaidova) [inline .github/workflows/llm_bench-python.yml:105]:
```suggestion
          python ./tools/llm_bench/benchmark.py -m tiny-random-qwen -d cpu -n 1 -f pt -ic 20
```

**Pull request merged**